plugins/toolkit/jetbrains-core/src/software/aws/toolkits/jetbrains/services/codemodernizer/CodeModernizerSession.kt

```diff
@@ -231,7 +231,7 @@ class CodeModernizerSession(
                 it.setRequestProperty(CodeWhispererCodeScanSession.SERVER_SIDE_ENCRYPTION_AWS_KMS_KEY_ID, kmsArn)
             }
         }
-            .connect { request ->
+            .connect { request -> // default connect timeout is 10s
```

Comment:
Question: Does this mean the upload stream closes after 10 seconds? Or aftert 10 seconds of inactivity?

Reply:
No: just tries for 10s to successfully, this hasn't caused any issues to our knowledge but just wanted to leave this comment here for the sake of info.